packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -261,7 +259,6 @@ export default class TradeStore extends BaseStore {
     barrier_count = 0;
     main_barrier: ChartBarrierStore | null = null;
     barriers: TBarriers = [];
-    hovered_barrier = '';
```

Comment:
Removing all logic with hovered barriers, because it was used in old Turbos. After Turbos revamp, there is no such functionality anymore